client/reader/following-manage/connected-subscription-list-item.jsx

```diff
@@ -51,9 +63,12 @@ class ConnectedSubscriptionListItem extends React.Component {
 				feed={ feed }
 				url={ url }
 				showEmailSettings={ showEmailSettings && ! isEmailBlocked }
+				isFollowing={ isFollowing }
 			/>
 		);
 	}
 }
 
-export default localize( connectSite( ConnectedSubscriptionListItem ) );
+export default connect( ( state, ownProps ) => ( {
+	isFollowing: isFollowingSelector( state, { feedId: ownProps.feedId, blogId: ownProps.siteId } ),
```

Comment:
Ah, interesting! I thought using the IDs might be more reliable. Why's it best to use URL in preference?